forms/views/admin/ab-testing-edit.php

```diff
@@ -1,4 +1,5 @@
 <?php
+include_once dirname(__DIR__, 3) . '/security-check.php';

```

Comment:
I think you would be better using `require` here instead of `include_once` or `require_once`, there is no harm in `security-check.php` being run more than once.

Reply:
Also it may just be better making this file reference relative versus `dirname(__DIR__, 3)`...I haven't tested it but sometimes that magic constant / function behaves differently depending on environment

Reply:
Hi Oliver, I changed to use relative path here, the reason for include_once is that we do not want to error out when the file can not be found. Using require will cause the application error out when it can not find the security-check.php